mirascope/core/mistral/_utils/_convert_message_params.py

```diff
@@ -1,23 +1,35 @@
 """Utility for converting `BaseMessageParam` to `ChatMessage`."""
 
-from mistralai.models.chat_completion import ChatMessage
+from mistralai.models import (
+    AssistantMessage,
+    SystemMessage,
+    ToolMessage,
+    UserMessage,
+)
 
 from ...base import BaseMessageParam
 
 
 def convert_message_params(
-    message_params: list[BaseMessageParam | ChatMessage],
-) -> list[ChatMessage]:
+    message_params: list[
+        BaseMessageParam | AssistantMessage | SystemMessage | ToolMessage | UserMessage
+    ],
+) -> list[BaseMessageParam]:
```

Comment:
this should be `list[SystemMessage | UserMessage | AssistantMessage | ToolMessage]`

Reply:
On the output?